frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -651,6 +662,10 @@ class Worksheet extends React.Component {
                             wsItems[focusIndex].mode === 'subworksheets_block')
                     ) {
                         // worksheet_item_interface and table_item_interface do the exact same thing anyway right now
+                        if (focusIndex === 0 && subFocusIndex === 0){
```

Comment:
This makes sure when the focus is on the very top, pressing up doesn't cycle around